src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="get"
+    data-qa="big"
+    class="search-bar search-bar__big">
```

Comment:
Need to check and fix it everywhere
<img width="1022" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/664e2dd1-0fdb-472f-a336-771e188cff81">